contact-us.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="CSS/contact-us style/form.css">
+    <title>Contact us</title>
+</head>
+<body>
+    <div class="form__container">
+        <form>

```

Comment:
A form should have an id generally speaking for easy recognition

Reply:
i see, i will fix that